security/core/src/it/java/org/seedstack/seed/security/internal/fixtures/TestActionResolver.java

```diff
@@ -22,34 +19,29 @@
 
 public class TestActionResolver implements CRUDActionResolver {
 
```

Comment:
I'd name the Class CrudActionResolver in order to be compliant to java standard name with acronyms and also to be flat with CrudSecurityIT. 
I also would rename CRUDAction to CrudAction for the same reasons.

Reply:
Agree. I suggested those names but it's better to keep CamelCase even for acryonyms.

Reply:
Agree, already commited the cangue :)